src/components/interaction-designer/block-types/Core_RunFlowBlock.vue

```diff
@@ -4,23 +4,31 @@
       {{'flow-builder.edit-block-type' | trans({block_type: trans(`flow-builder.${block.type}`)})}}
     </h3>
     <fieldset :disabled="!isEditable">
-      <block-name-editor :block="block" />
-      <block-label-editor :block="block" />
-      <block-semantic-label-editor :block="block" />
+      <validation-message :message-key="`block/${block.uuid}/.name`" #input-control="{ isValid: isNameValid }">
+        <block-name-editor :block="block" :validationState="!isNameValid" />
+      </validation-message>
+      <validation-message :message-key="`block/${block.uuid}/.label`" #input-control="{ isValid: isLabelValid }">
+        <block-label-editor :block="block" :validationState="!isLabelValid"/>
+      </validation-message>
+      <validation-message :message-key="`block/${block.uuid}/.semantic_label`" #input-control="{ isValid: isSemanticLabelValid }">
+        <block-semantic-label-editor :block="block" :validationState="!isSemanticLabelValid" />
+      </validation-message>
 
-      <div class="form-group">
-        <label>{{ 'flow-builder.destination-flow' | trans }}</label>
+      <validation-message :message-key="`block/${block.uuid}/.config.flow_id`" #input-control="{ isValid: isFlowIdValid }">
+        <div class="form-group">
+          <label>{{ 'flow-builder.destination-flow' | trans }}</label>
+          <select class="form-control" v-model="destinationFlowId" :class="{ 'is-invalid': isFlowIdValid === false }">
```

Comment:
`isFlowIdValid` is a boolean, so let's use `!isFlowIdValid` instead of `isFlowIdValid === false`

Reply:
Interesting thing I noticed when I tried making this change. It does not work with !isFlowIdValid, but works with just isFlowIdValid. I noticed this in other places too!  I guess the value coming in for isValid is actually the value for isInvalid